### PR TITLE
Add Donut component

### PR DESCRIPTION
**Pull request by HorstFrank**

### CODE REVIEW

**Review by sweisz — COMMENTED** [review]
Great 🍩

File: components/Donut/Donut.module.css [hunk 1]
```diff
@@ -3,12 +3,25 @@
...
   letter-spacing: 0.1em;
   padding-top: 2em;
 }
 
 .label {
-  font-size: 0.9rem;
-  margin-left: 0.5rem;
+  /* font-size: 0.9rem;
+  margin-left: 0.5rem; */
+  /* fill: var(--icon); */
```
Comment (mewiemers) [inline components/Donut/Donut.module.css:14]: oh soo many comments.. you need this anymore?

File: components/Donut/Donut.module.css [hunk 2]
```diff
@@ -3,12 +3,25 @@
...
+
+/*
+dy
+Shifts the text position vertically from a previous text element.
+
+class, style, font-family, font-size, font-size-adjust, font-stretch, font-style, font-variant, font-weight
+
+
+
+*/
```
Comment (mewiemers) [inline components/Donut/Donut.module.css:27]: the same here

File: components/Donut/Donut.tsx [hunk 3]
```diff
@@ -4,12 +4,32 @@ import Icon from "../icons/Icons";
...
+  //     value: data[key],
+  //     start: data[key],
+  //     end: data[key],
+  //   };
+  //   // return {
+  //   //   ...data[key],
+  //   //   start: 2,
+  //   //   end: 2,
+  //   // };
+  // });
```
Comment (mewiemers) [inline components/Donut/Donut.tsx:32]: or here

**Review by mewiemers — COMMENTED** [review]
fine job, just a few comments on your comments ;-) and a tiny merge conflict i see ;-)

**Pull request closed without merging**